cmd/utils/cmd.go

```diff
@@ -173,6 +174,17 @@ func ImportChain(chain *core.BlockChain, fn string) error {
 			return err
 		}
 	}
+	cpuProfile, err := os.Create("cpu.out")
+	if err != nil {
+		return fmt.Errorf("Error creating CPU profile: %v", err)
+	}
+	defer cpuProfile.Close()
+	err = pprof.StartCPUProfile(cpuProfile)
+	if err != nil {
+		return fmt.Errorf("Error starting CPU profile: %v", err)
+	}
+	defer pprof.StopCPUProfile()
+
```

Comment:
the replay code has been merged into this, it needs to be un-merged before everything is merged into `beverly-hills-head`.